src/pages/AdminTemplates/AdminTemplates.tsx

```diff
@@ -0,0 +1,154 @@
+import classNames from "classnames";
+import React, { useState } from "react";
+
+import { DEFAULT_MAP_BACKGROUND } from "settings";
+
+import { Room } from "types/rooms";
+
+import { PartyMapContainer } from "pages/Account/Venue/VenueMapEdition";
+
+import "./AdminTemplates.scss";
+
+interface TemplateType {
+  title: string;
+  description: string;
+  background: string;
+}
+
+interface RecipeType {
+  title: string;
+  rooms: Room[];
+}
+
+const templates: TemplateType[] = [
+  {
+    title: "Title",
+    description: "Description",
+    background: DEFAULT_MAP_BACKGROUND,
+  },
+  {
+    title: "Title",
+    description: "Description",
+    background: DEFAULT_MAP_BACKGROUND,
+  },
+  {
+    title: "Title",
+    description: "Description",
+    background: DEFAULT_MAP_BACKGROUND,
+  },
+  {
+    title: "Title",
+    description: "Description",
+    background: DEFAULT_MAP_BACKGROUND,
+  },
+];
+
+const recipes: RecipeType[] = [
+  { title: "Recipe 1", rooms: [] },
+  { title: "Recipe 2", rooms: [] },
+  { title: "Recipe 3", rooms: [] },
+  { title: "Blank Canvas", rooms: [] },
+];
+
+export const AdminTemplates: React.FC = () => {
+  const [selectedTemplate, setSelectedTemplate] = useState<TemplateType>();
+  const [selectedRecipe, setSelectedRecipe] = useState<RecipeType>(recipes[0]);
+  const [isLoading, setLoading] = useState<boolean>(false);
+
+  const selectTemplate = (template: TemplateType) => {
+    if (isLoading) return;
+
+    setLoading(true);
+
+    setTimeout(() => {
+      setSelectedTemplate(template);
+      setLoading(false);
+    }, 2000);
+  };
```

Comment:
`useCallback`

Extract the timeout into a `const` (probably in `src/settings`) or reuse a relevant one that already exists there rather than using a 'magic number' here.

We probably want to be 'triggering' the `setTimeout` within a `useEffect` and also ensuring that we return a cleanup function that cancels the timeout.

Reply:
Also.. what's the intent/purpose of this code in general? It looks like we're introducing an arbitrary delay for... no reason?